src/Perpetuum/Services/Channels/ChatCommands/AdminCommandHandlers.cs

```diff
@@ -170,13 +170,21 @@ public static void ShutdownCancel(AdminCommandData data)
         [ChatCommand("JumpTo")]
         public static void JumpTo(AdminCommandData data)
         {
-            bool err = false; //TODO this only throws on bad last-arg!
-            err = !int.TryParse(data.Command.Args[0], out int zone);
-            err = !int.TryParse(data.Command.Args[1], out int x);
-            err = !int.TryParse(data.Command.Args[2], out int y);
-            if (err)
+            int zone;
```

Comment:
Here is another quick thing to add in as these get refactored:

Anytime a zone parameter is being checked, you can also call the helper:
```
CheckZoneId(data, zoneId);
```
Which will check against the set of active zones, throwing if the id is not for an active/valid zone.